resource-watch/resource-watch
# Land Watch landing page

## Overview

This PR adds a landing page for the Land Watch content.

The page is structured to allow for content that has a maximum width as well as content that can expand to the entire width of the window.

### Notes

In order to add the dashboard to the dashboards dropdown, we'd need to add it to the production api. I have not done this.

### Screenshots/Demo

![localhost_3000_dashboards_land-watch](https://user-images.githubusercontent.com/2442245/131405931-53fddf68-909e-4587-8369-0a37af005c1c.png)

## Testing instructions
 * ensure that your env includes `NEXT_PUBLIC_FEATURE_FLAG_LAND_WATCH=true`.
 * navigate to /dashboards/land-watch
 * it should load and look like the screenshot above
 * change the feature-flag env variable to `false`, restart the dev server
 * verify that the page is no longer visible

## Checklist before submitting
